src/js/components/charts/TasksChart.js

```diff
@@ -31,26 +35,31 @@ var TasksChart = React.createClass({
     var newTasks = this.getTasks(nextProps.tasks);
 
     // If equal, do not update
-    return !_.isEqual(previousTasks, newTasks);
+    return !Object.keys(previousTasks).reduce(function (isEqual, key) {
```

Comment:
do we even want to do this? we're replacing this anyways right? we can leave it here since it's more work to remove it